packages/frontend/src/Layout/Footer/Footer.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
```

Comment:
Importuj tak w każdym komponencie:  
import \* as React from 'react'  
  
https://twitter.com/dan_abramov/status/1308739731551858689

Reply:
teraz już jest chyba nawet niepotrzebny import jeżeli jest nieużywany. Wkażdym innym wypadku szedłbym w destrukturyzacje

Reply:
Od reakta 17. Nie pamoetam ktora wersje mamy

Reply:
no to my 17 mamy napewno

Reply:
Mamy 17.01. Więc jeżeli nie używasz, żadnych hooków to nie musisz importować reacta